ducktape/cluster/json.py

```diff
@@ -23,11 +23,8 @@ class JsonCluster(Cluster):
     An implementation of Cluster that uses static settings specified in a cluster file.
     """
 
-    def __init__(self, cluster_json=None):
+    def __init__(self, cluster_json, *args, **kwargs):
         super(JsonCluster, self).__init__()
-        if cluster_json is None:
```

Comment:
This branch was intended to support manually defined clusters, i.e. if you have a set of nodes and just want to maintain a list of them manually. It's important to support clusters like this since not everyone's testbeds will be managed by Vagrant.